pkg/apis/config/feature_flags.go

```diff
@@ -358,3 +364,16 @@ func GetVerificationNoMatchPolicy(ctx context.Context) string {
 func IsSpireEnabled(ctx context.Context) bool {
 	return FromContextOrDefaults(ctx).FeatureFlags.EnforceNonfalsifiability == EnforceNonfalsifiabilityWithSpire
 }
+
+type PerFeatureFlag struct {
```

Comment:
Can you add a TODO here as the struct is currently not used

Reply:
Oh, sorry. I mean adding this tracking issue as comment in the codebase. As a developer, I could get confused by an added but never used struct (unless I missed the usage of it)

Reply:
Cool thanks. Done. Yeah that issue came into place just today.